### PR TITLE
Z3 solver, optimization edition.

### PR DESCRIPTION
**Pull request by sheganinans**
This may not be the right way to do things. Suggestions are appreciated.

### CODE REVIEW

File: pkgs/applications/science/logic/z3_opt/default.nix [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    cp ../src/api/c++/z3*.h   $out/include
+    cp z3                     $out/bin
+    cp libz3${soext}          $out/lib
+    cp libz3${soext}          $out/lib/${python.libPrefix}/site-packages
+    cp z3*.pyc                $out/lib/${python.libPrefix}/site-packages
+    cp ../src/api/python/*.py $out/lib/${python.libPrefix}/site-packages
+  '';
+
+  meta = {
+    description = "A high-performance theorem prover and SMT solver, optimization edition.";
```
Comment (jagajaga) [inline pkgs/applications/science/logic/z3_opt/default.nix]: Remove dot at the end please

**Comment by sheganinans** [comment]
It seems like I need to add myself to the maintainers. Where is that file?

**Comment by iyzsong** [comment]
It's at ```lib/maintainers.nix```.

**Comment by sheganinans** [comment]
Thanks! Ok, merged and squashed.

**Pull request merged**